assets/src/application/ui/display/icons/HelpIcon/style.scss

```diff
@@ -0,0 +1,4 @@
+.ee-help-icon {
+	color: var(--ee-color-primary-low-contrast);
+	margin: 0 var(--ee-margin-tiny);
```

Comment:
I was able to reproduce the icon w/o the color using `yarn prod`:
![image](https://user-images.githubusercontent.com/1477453/85567091-3eba2880-b639-11ea-8705-ebf237b2945e.png)
***
after extracting that icon styles it worked on prod the same as on dev

![image](https://user-images.githubusercontent.com/1477453/85567596-ad978180-b639-11ea-851d-016c41384a8e.png)